languages/csharp/exercises/concept/numbers-floating-point/.docs/after.md

```diff
@@ -1,4 +1,4 @@
-There are three floating-point types in C#: `float`, `double` and `decimal`. The most commonly used type is `double`, whereas `decimal` is normally used when working with monetary data.
+There are three floating-point types in C#: `float`, `double` and `decimal`. The most commonly used type is `double`, whereas `decimal` is normally used when working with monetary data. A `double` is defined as `2.45` or `2.45d`, a `float` as `2.45f` and a decimal as `2.45m`.
```

Comment:
```suggestion
There are three floating-point types in C#: `float`, `double` and `decimal`. The most commonly used type is `double`, whereas `decimal` is normally used when working with monetary data. A `double` is defined with the syntax `2.45` or `2.45d`, a `float` with `2.45f` and a decimal with `2.45m`.
```

Is "defined" the right word? Rather than assigned/created/initiated, etc?

Reply:
Ah, it should be "declared"